apps/src/templates/instructions/TopInstructionsCSF.jsx

```diff
@@ -324,6 +331,9 @@ var TopInstructions = React.createClass({
    * collapsed
    */
   getMinHeight(collapsed=this.props.collapsed) {
+    if (this.refs.containedLevel) {
+      return $('#containedLevel0').outerHeight() + 2 * CONTAINED_LEVEL_PADDING;
```

Comment:
I had to remove the `minHeight` previously applied to the `ContainedLevel` component to make it work, but it doesn't look like that was necessary in the first place. Done.